docs/source/development.rst

```diff
@@ -106,14 +106,164 @@ are making:
 
     $ git remote add upstream https://github.com/ga4gh/server.git
 
+All development is done against the ``develop`` branch.  The ``master``
+branch is meant to be kept stable since it is the branch releases are
+based on -- don't touch it!  These are the two mainline branches.
+Our branching model is loosely based on the one described
+`<here http://nvie.com/posts/a-successful-git-branching-model/>`_.
+
+All development should be done in a topic branch.  That is, a branch
+that the developer creates him or herself.  These steps will create
+a topic branch (assuming that ``upstream`` points to 
+``https://github.com/ga4gh/server.git``) and ``origin`` points to 
+``https://github.com/YOUR_GITHUB_USERNAME/server.git``, which can be
+verified by running ``git remote -v``):
 
-**TODO**
+.. code-block:: bash
+
+    $ git fetch --all
+    $ git checkout develop
```

Comment:
There are two differences between your approach and my approach:
1) You fetch only updates from the upstream repo, whereas I fetch updates from all remote repos.
2) You don't create a local `develop` branch, whereas I do.

Advantages of my approach:
- no chance of a new user screwing up the fetch command by mistyping, or fetching the wrong remote repo... just fetch 'em all!  (fetch is a pretty easy command for newbies to screw up http://stackoverflow.com/questions/21537244/differences-between-git-fetch-and-git-fetch-origin-master )
- having an updated local `develop` branch saves a little typing when one wants to reference it

Disadvantages of my approach:
- little slower to fetch all repos (not really a problem in practice)
- need to maintain a local copy of develop (not a problem with my alias below)